### ISSUE TITLE
[GOG, Linux] Incorrect strings throughout the UI

### ISSUE THREAD

**Issue by AvinashReddy3108** [wolfetplayer/RealRTCW#98]
# Machine info
OS: Arch Linux x86_64
Kernel: 6.4.12-zen1-1.1-zen
CPU: Intel i3-4130 (4) @ 3.400GHz
GPU: NVIDIA GeForce GT 1030

# Game info
Game version: `2.0.0.2 (GOG)`, `setup_return_to_castle_wolfenstein_2.0.0.2.exe`
RealRTCW assets version: `4.0.13 (ModDB)`, `realrtcw4.0.13.zip`
RealRTCW binary: [Linux, v4.0-nosteam](https://github.com/wolfetplayer/RealRTCW/releases/download/4.0/RealRTCW-4.0-nosteam-linux.zip)

# Game Files Structure
_Includes the SHA256 checksum of the files._
```text
 <GAME DIRECTORY>
  | <GAME DIRECTORY>/main
  |  |-- autoexec.cfg [2a48c3b814590a391ad5746fd61515e2a0dd87fa8cae20382ca87f2e8d8f0f04]
  |  |-- cgame.sp.x86_64.so [ac3962a5997d4ac32232fcc431b1c6b183da8d2e82b92d487b07bbd7fe0df4c7]
  |  |-- mp_bin.pk3 [bb6ee2302de55cd35a3145b790e43958fa92a98d3494b9fa96b457d030b0c05d]
  |  |-- mp_pak0.pk3 [c0e090e7b4245313bea64b5fb20b24d69786e593716cfe6bf85ee9b4f3c794b3]
  |  |-- mp_pak1.pk3 [248ff7c6a4c12f3fc08e0bb73a0602ffdb2e31c4a4c50929d76fa5f814cbf974]
  |  |-- mp_pak2.pk3 [0a9142be2539427189cc9b31d5af188298ce3e4c8f06d752e1809d22046b7b03]
  |  |-- mp_pak3.pk3 [ad7705f8c05154f6e009f29d1a7e3eb5bf9cb26cb9035d9c55640ec02cb14692]
  |  |-- mp_pak4.pk3 [61bf3e778d2777895ce7c0c7afe6df97f2521a3e5ed054c04346ea5ccdcac0aa]
  |  |-- mp_pak5.pk3 [8defe8227bd345c485bac98f8c1191c2360d0c784953e9c38a81d245eb1901b9]
  |  |-- mp_pakmaps0.pk3 [2e9cfdc505587dc7e7059e3188b9a1b4ceba75decfa7578c3b2a73dbb0d94a93]
  |  |-- mp_pakmaps1.pk3 [a907dc0caf8ab4284dc8ba8de04a11ec8472a50f861998994c834ff1a4781d9a]
  |  |-- mp_pakmaps2.pk3 [d2cc0a7a762ad5f1ec3523ff79798cd7d732ffe8f29e6ac7df66f74319e85eb6]
  |  |-- mp_pakmaps3.pk3 [524ecd9053d4c8f8e27d4bc4c04504fc1a1f3a29dd8299c8b38909db3c51e6ac]
  |  |-- mp_pakmaps4.pk3 [1357a12e57634dac59cbe70a5984af85a43e7db5aff930d05c8ee42dfcefded8]
  |  |-- mp_pakmaps5.pk3 [85cc0e8358a26601211625a9600ca47c126e4a38cca0bac60d7465ffb5ccadfe]
  |  |-- mp_pakmaps6.pk3 [fcdb3754ca4e9f60b3086c8461b26160dd8de7f5e94c04100a2eb6f9519abca2]
  |  |-- pak0.pk3 [1dfcfc3187ba09fe32cd8a7921a09cd15dd789816186c89b11e5e0d954c55e6c]
  |  |-- qagame.sp.x86_64.so [23d95260bd3255a92e16462aa9fee0e7000bca9a493462526cd00fa678970b86]
  |  |-- realrtcwdefault.cfg [dc06abede1e57a0a02047c35320a9b33f5b04ac62a7cb3903201d26e4588bee0]
  |  |-- rotate.cfg [b0d5fa3f029167d73eddabec562938e6ee8d97b2a7f5215dde66525a574b40d5]
  |  | <GAME DIRECTORY>/main/scripts
  |  |  |-- translation.cfg [301670fbd41eb6797b02cc5ba8ae5e221eeaef5de3280e6e2db390c7a29ddc8f]
  |  |-- sp_pak1.pk3 [66785d612f88a19c3bb5fd24f84d89b240375a28550a79039dfb6afae865ec9a]
  |  |-- sp_pak2.pk3 [eca64ea0fda6edd653865e2eb768f152d38abc8b89941aa954ed74fa36928796]
  |  |-- sp_pak3.pk3 [936437413956f88b9e5d548ccab9b51b8c8b9754f8f8e13ee7b910813a24d0f9]
  |  |-- sp_pak4.pk3 [5532801005eaf2d74fe9098dc638abdb8284225c293cee3573ffa9b6c2edd8fc]
  |  |-- ui.sp.x86_64.so [9786599bf67daf75deaf555af212c08b82c7504db222b8240876418dc1c64435]
  |  |-- xbox.cfg [1de788b4214f35b0cd6f22c75e09f6beb2960c57bb1e5d4f0b56a767a09c38f0]
  |  |-- xbox_cinematic.cfg [975633bbd56d4d4827cc955c6121b3d22c5dfecf2b42f000ada64afe649ddf0d]
  |  |-- z_realrtcw_localization.pk3 [8c6f467d65689bbe8abc7717db99c8cf4e7c1fe9ea49b3b64a2ecead13e00b02]
  |  |-- z_realrtcw_maps.pk3 [1a39cd259543849041ecc8c1bfcbc984f23ff9e943720114fde616955e8ce2de]
  |  |-- z_realrtcw_models_characters.pk3 [8451c15e918d5f49fa320d052ade873df70d7965c21af7c0d24f75a7e7528615]
  |  |-- z_realrtcw_models.pk3 [23ca646c371eb350d14ea4c3f363ce9d45025d796628b65ecec8448392ced07d]
  |  |-- z_realrtcw_models_weapons.pk3 [d0daf19f8898c95bb39b5e4454aca28860df9663d74c34de6a43b6e8c307c484]
  |  |-- z_realrtcw_sounds.pk3 [551ba8b319bdf5f0cdcd95c556df9f7fc16d9cbdd3b5e0c2082b499e11b3982c]
  |  |-- z_realrtcw_textures.pk3 [6b41bc587da76d4d1b212fdbe50861757b424c76cec2e295b227473a75beb2c1]
  |  |-- z_trainingmap.pk3 [43626dded707144763be7eff8d6f9b8b968071698280ad0aea2c4f1503e55c03]
  |  |-- z_zperson.pk3 [47f0e5df18bbcb4b810e01dcf3fb6b287d249fae79b3559463fc817e94f44944]
  |  |-- z_zrealrtcw_ui.pk3 [41cf0f225123d18c05e3b1aaae8f4e80017362eddbe52b6a8656d587623c3742]
  |  |-- z_zzrealrtcw_scripts.pk3 [d448ccf0ea1222e1fbfee5b1e30e8bde1f3203cd91220022949f64adc1e53f37]
  |-- RealRTCW.x86_64 [fd43fc8dff67de96731af60f35ffcde57f137d59938f5215161b712174d4e2a9]
  |-- renderer_sp_opengl1_x86_64.so [d8760164f72ef873dfe1f49ce338d3e9169c2c60b9408b3d7ebe256f4be26bf6]
  |-- renderer_sp_rend2_x86_64.so [63e6cb66dd733b63e82a65b65e76ce554579bc37db4356c8f4d696d7d36d62e6]
```
_TL;DR: It's the GOG Game Files + [patch-data-141](https://github.com/iortcw/iortcw/releases/download/1.51c/patch-data-141.zip) + [RealRTCW assets from ModDB](https://www.moddb.com/mods/realrtcw-realism-mod/downloads/realrtcw-40) + [RealRTCW Linux Binary](https://github.com/wolfetplayer/RealRTCW/releases/download/4.0/RealRTCW-4.0-nosteam-linux.zip)_

# Issue
There are various instances of incorrect strings being used throughout the UI, see Screenshots below..

# Screenshots
![Screenshot_20230903_232146](https://github.com/wolfetplayer/RealRTCW/assets/27774996/159619dc-f4c6-4511-949d-6f8130bb771e)
_I'm pretty sure the "Exit not yet available" is not a valid value for these settings.._

**Comment by wolfetplayer** [comment]
If you compiling the master branch - it is expected. Master branch is heavily WIP and does not work without proper game archives which are currently not available.

**Comment by wolfetplayer** [comment]
Oh. Just read that you were using linux build from releases... Thats correct?

**Comment by inshy2** [comment]
Should we wait and hope that problems with the correct display of text in menus and messages will be fixed?
Please, fix the Linux version.

**Comment by wolfetplayer** [comment]
Its just an outdated build, the code is just fine. I will ask someone with linux experience to recompile it and will upload it.
Sorry about that but im 0 at linux stuff.

**Comment by MatiasRepetto** [comment]
> How to download warbell-temp branch? I will build it by myself

https://github.com/wolfetplayer/RealRTCW/archive/refs/heads/warbell-temp.zip download this and compile as always
